src/sql/workbench/browser/modal/modal.ts

```diff
@@ -172,6 +172,8 @@ export abstract class Modal extends Disposable implements IThemable {
 		}
 
 		this._bodyContainer = DOM.$(`.${builderClass}`, { role: 'dialog', 'aria-label': this._title });
+		const top = this.layoutService.getTitleBarOffset();
```

Comment:
This is the real change, we need to adjust the top based on the title bar. The rest of the changes are converting layoutservice to workbenchlayoutservice to gain access to this method.